ops/terraform/envs/dev/data.tf

```diff
@@ -0,0 +1,3 @@
+data "azurerm_resource_group" "dev" {
+  name = "reportvision-${local.environment}-rg"
```

Comment:
@zdeveloper heres the name change for the dev resource group and I also name change all the other resources.